ra/registration-authority.go

```diff
@@ -40,14 +41,16 @@ type RegistrationAuthorityImpl struct {
 	log         *blog.AuditLogger
 	// How long before a newly created authorization expires.
 	authorizationLifetime time.Duration
+	rlPolicies            map[string]cmd.RateLimitPolicy
 }
 
 // NewRegistrationAuthorityImpl constructs a new RA object.
-func NewRegistrationAuthorityImpl(clk clock.Clock, logger *blog.AuditLogger) RegistrationAuthorityImpl {
+func NewRegistrationAuthorityImpl(clk clock.Clock, logger *blog.AuditLogger, policies map[string]cmd.RateLimitPolicy) RegistrationAuthorityImpl {
```

Comment:
Why a map instead of a struct? This seems like it would cause a lot of annoying bugs.